content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -188,27 +227,27 @@ const { SemanticAttributes } = require('@opentelemetry/semantic-conventions');
 Finally, you can update your file to include semantic attributes:
 
 ```javascript
-function doWork(parent) {
-  const ctx = opentelemetry.trace.setSpan(
-    opentelemetry.context.active(),
-    parent
-  );
+const doWork = () => {
   const span = tracer.startSpan(
-    'doWork',
+    'app.new-span',
     { attributes: { [SemanticAttributes.CODE_FUNCTION]: 'doWork' } },
     ctx
   );
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {
-    // empty
-  }
+
+  // Add an attribute to the same span later on
   span.setAttribute(SemanticAttributes.CODE_FILEPATH, __filename);
+
+  // Make sure to end the span if you're done with it!
   span.end();
 }
 ```
 
 ## Span events
 
-An event is a human-readable message attached to a span that represents "something happening" during its lifetime. You can think of it like a primitive log.
+A [Span Event](/docs/concepts/signals/traces/#span-events) is a human-readable
```

Comment:
> IMO "something happening" is too general. I prefer to think of events as an activity that is tracked as a single point in time rather than a start time and and end time. For example, a page load may take time and be represented by a span, but the moment a page becomes interactive is a singular point in time and may be represented as a span event.

Reply:
I will reword this a bit and then open a separate PR that gives a motivating wordy example in the span event section in language-agnostic conceptual docs.